plant-and-cell-physiology.csl

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" et-al-min="6" et-al-use-first="6" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Plant and Cell Physiology</title>
```

Comment:
@damnation333, it looks like this title uses an ampersand instead of "and". You can use XML-escaped ampersands with `&amp;`.